subprojects/testing-base/src/main/java/org/gradle/api/tasks/testing/JUnitXmlReport.java

```diff
@@ -57,4 +58,21 @@ public interface JUnitXmlReport extends DirectoryReport {
     @Input
     Property<Boolean> getMergeReruns();
 
+    /**
+     * Decide to include or omit the system out log in the XML report. The default behavior is to have it.
+     *
+     * @since 8.1
```

Comment:
Please change these to 8.2 to target the next release.

Reply:
sorry for the lack of activity. I needed to focus on sth else in recent weeks. I'll continue to work on it this week (probably during the weekend)

Reply:
Glad to hear it!  If it takes a while to get back to this, just check https://gradle.org/release-candidate/ and change these `@since` tags to target the next minor version above whatever the current RC is.